src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  margin: 120px 0 110px;
+
+  @include onTablet {
+    margin-bottom: 96px;
+  }
+
+  @include onDesktop {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    @include title;
+    margin: 0 0 32px;
+    text-align: left;
+
+    @include onTablet {
+      font-size: 40px;
+      line-height: 40px;
+    }
+
+    @include onDesktop {
+      font-size: 44px;
+      line-height: 48px;
+    }
+  }
+
+  &__bullet-points {
+    margin-bottom: 40px;
+
+    &--text {
+      margin: 0;
```

Comment:
Consider using a variable for the font-weight to make it reusable and easily maintainable. You can create a variable like $font-weight-bold: 700; in your variables.scss file and use it here. 🎯